mabl snippets/sendEnter.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Sends an enter key press to an element specified using a selector in
+ * the enterTargetSelector variable.
```

Comment:
Might be nice to separate out the prerequisite variables. Maybe make a section for the variables and what to put in them.

```
/*
 * required mabl variables:
 *   enterTargetSelector - an XPath/css selector to locate the target element
 * ...
 */
```

Reply:
Great idea.